packages/fxa-settings/src/pages/Signin/container.tsx

```diff
@@ -131,26 +133,31 @@ const SigninContainer = ({
           accountStatus.hasLinkedAccount === undefined ||
           accountStatus.hasPassword === undefined
         ) {
-          // TODO: error handling for this (and in SignUp)
-          const { exists, hasLinkedAccount, hasPassword } =
-            await authClient.accountStatusByEmail(email, {
-              thirdPartyAuthStatus: true,
-            });
-          if (!exists) {
-            // For now, just pass back emailStatusChecked. When we convert the Index page
-            // we'll want to read from router state.
-            navigate(`/signup?email=${email}&emailStatusChecked=true`);
-            // TODO: Probably move this to the Index page onsubmit once
-            // the index page is converted to React, we need to run it in
-            // signup and signin for Sync
-          } else {
-            setAccountStatus({
-              hasLinkedAccount,
-              hasPassword,
-            });
-            if (isSyncWebChannel) {
-              firefox.fxaCanLinkAccount({ email: queryParamModel.email });
+          try {
+            const { exists, hasLinkedAccount, hasPassword } =
+              await authClient.accountStatusByEmail(email, {
+                thirdPartyAuthStatus: true,
+              });
+            if (!exists) {
+              // For now, just pass back emailStatusChecked. When we convert the Index page
+              // we'll want to read from router state.
+              navigate(`/signup?email=${email}&emailStatusChecked=true`);
+              // TODO: Probably move this to the Index page onsubmit once
+              // the index page is converted to React, we need to run it in
+              // signup and signin for Sync
+            } else {
+              setAccountStatus({
+                hasLinkedAccount,
+                hasPassword,
+              });
+              if (isSyncWebChannel) {
+                firefox.fxaCanLinkAccount({ email: queryParamModel.email });
+              }
             }
+          } catch (error) {
+            // if we aren't able to check account status, redirect to email first
+            // TODO: send back with error message to display on email-first?
```

Comment:
Thanks for adding the try/catch! I don't think we should display an error message, once they re-renter their email the endpoint will be hit again and that'll show an error message if it errors again. I suppose we could add a `prefillEmail` param here with the email but it's an edgecase so 🤷‍♀️